conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -127,6 +127,49 @@ TEST_F(
   run_device_hierarchy_child_process();
 }
 
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDeviceWhenRetrievingSubDevicePropertiesThenValidPropertiesAreReturned) {
+  for (auto device : devices) {
+    auto device_properties = lzt::get_sysman_device_properties(device);
+    uint32_t sub_devices_count = device_properties.numSubdevices;
+
+    if (sub_devices_count > 0) {
+      uint32_t num_subdevices = 0;
+      auto sub_device_properties =
+          lzt::get_sysman_subdevice_properties(device, num_subdevices);
+      EXPECT_EQ(sub_devices_count, num_subdevices);
+      for (uint32_t sub_device_index = 0; sub_device_index < num_subdevices;
+           sub_device_index++) {
+        EXPECT_LT(sub_device_properties[sub_device_index].subdeviceId,
+                  num_subdevices);
+        EXPECT_GT(sub_device_properties[sub_device_index].uuid.id[0], 0);
+      }
```

Comment:
Query: Should other fields (like _sType_, _pNext_) be not checked for each of the sub_device's properties?

Reply:
I think there's no need to check for stype since for sub devices there is no place where it has been used. and pNext is optional.